tests/test_req_vars.py

```diff
@@ -647,3 +647,68 @@ def _test_error_details(self, headers, attr_name,
         except error_type as ex:
             self.assertEqual(ex.title, title)
             self.assertEqual(ex.description, description)
+
```

Comment:
Looks like all the new tests have `protocol='HTTP/1.0'`... I'm not seeing any for checking with the default `'HTTP/1.1'` protocol?